core/js/typespecs/app_types.ts

```diff
@@ -1200,6 +1222,31 @@ export const patchAISecretSchema = z.object({
   secret: z.string().nullish(),
 });
 
+export const createEnvVarSchema = z.object({
+  object_type: envVarObjectTypeEnum,
+  object_id: z
+    .string()
+    .uuid()
+    .describe("The id of the object the environment variable is scoped for"),
+  name: z.string().describe("The name of the environment variable"),
+  value: z
+    .string()
+    .nullish()
+    .describe(
+      "The value of the environment variable. Will be encrypted at rest.",
+    ),
+});
+
+export const patchEnvVarSchema = z.object({
+  name: z.string().nullish().describe("The name of the environment variable"),
```

Comment:
Same here. Would prefer to pick from existing schemas instead of copy-pasting.

Reply:
done